qstnrs/qstnrs/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import patterns, include, url
+from django.contrib import admin
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^qstnrs/', include('questionnaire.urls')),
+    url(r'^admin/', include(admin.site.urls)),
```

Comment:
I'm sorry, I was confused by the multiple admin.autodiscover(). I think it should be only in one place. You have it in the app urls as well. You should drop it since it might not work if django admin app is not installed in your project.